src/main.rs

```diff
@@ -79,6 +79,8 @@ fn main() {
                             let (lhs, rhs) = two_param(&mut stack_data);
                             lhs - rhs
                         }
+                        // Ok(Operator::Absolute) => Abs::abs(stack_data.pop().unwrap()),
```

Comment:
remove this line.

Reply:
Well, yes, remove this line but, this makes me think... you always copy the image when applying an operator, right? It might be interesting to **not** do that! Maybe, make your input mutable, load it in f64, then apply all operator on it? Thoughts?

Reply:
From how I implement it I only have a maximum of three image at the same time in memory no ?
The `lhs`, `rhs`, and the results and after that `lhs` and `rhs` will not used used how rust is behaving with with?
The operator do not provides a way to pass a ref `fn add (self, rhs: Self::Output) -> Self::Output`.
Also if I want to have a similar implementation as mrtrix I'll need a kind of reference for the loaded images to not load it N times, load it one time and use it where it's needed.

Reply:
I use `mapv_into` it seems to be an inplace operation return the ndarray itself, I could use this to implement all the unary f64 functions.